### ISSUE TITLE
graph template

### ISSUE THREAD

**Issue by rom1701** [shinken-monitoring/mod-webui#387]
Hi,

If I use a template for my graph, the buttons 4 hours / 1 days / 1 week / 1 month does not work only for this graph. 
I don't see GET request when i click.

Have you the same problem?

**Comment by mohierf** [comment]
You are right ... I just observed this also. It seems that the `ui-graphite` module does only provide one URL to the WebUI ... I will try to fix this!

**Comment by kernhsab** [comment]
Hi, i've the same issue on my fresh shinken install with webui2 and ui-graphite2. 
Maybe is there a way to insert the graph_start and graph_end variable in the graph template?

Thanks for your help
